requirements.txt

```diff
@@ -1,10 +1,10 @@
 # Python Requirements File
 
 # dbt packages
-dbt-bigquery==1.6.1
-dbt-core==1.6.0
-dbt-extractor==0.4.1
-dbt-snowflake==1.6.0
+dbt-bigquery==1.7.6
+dbt-core==1.7.9
+dbt-snowflake==1.7.2
+dbt-redshift==1.7.4
```

Comment:
Did you check out the release notes for these? Anything we should be aware of?

Reply:
Here are the minor release notes from [dbt-bigquery](https://github.com/dbt-labs/dbt-bigquery/releases/tag/v1.7.0) and here are the notes from [dbt-snowflake](https://github.com/dbt-labs/dbt-snowflake/releases/tag/v1.7.0). Patches seem to be mostly bug fixes.

I don't think it's anything that affects us too much, but nice to see continued development and addition of features.